Assets/Scripts/AppLovinController.cs

```diff
@@ -0,0 +1,324 @@
+﻿using System;
+using DefaultNamespace;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class AppLovinController : MonoBehaviour
+{
+    [SerializeField] private TextAsset keysFile;
+    private string MaxSdkKey;
+    private string InterstitialAdUnitId;
+    private string RewardedAdUnitId;
+    private string BannerAdUnitId;
+
+    public Button showInterstitialButton;
+    public Button showRewardedButton;
+    public Button showBannerButton;
+    public Button mediationDebuggerButton;
+    public Button shareScreenshotFBButton;
+    public Text interstitialStatusText;
+    public Text rewardedStatusText;
+
+    private bool isBannerShowing;
+    private bool isMRecShowing;
+
+    private int interstitialRetryAttempt;
```

Comment:
You could have a class managing a UI and the other handling the event to allow a better versatility